docs/local_resource.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Run Local and/or Occasional Workflows with Local Resource
+layout: docs
+---
+A Local Resource works like any other resource in your sidebar (which are most
+often some combination of Docker Build instructions and K8s Yaml)--encompasses
+a unit of work, and execute in response to file changes, or
+[manually](https://docs.tilt.dev/manual_update_control.html). For your resource
+`MyGreatService`, when one of its file dependencies changes, the work it executes
+is to build a Docker image and deploy some k8s yaml; for a Local Resource, it's
+to execute an arbitrary command on your local filesystem.
+
+# Run commands locally on file change
+
+Tilt is designed to make it easy to build images, and to sync files/run commands
```

Comment:
This sentence kind of makes it sound like Local Resource is some fringe feature that tilt is not designed to make easy.

Maybe something like "Most uses of Tilt focus on easily and quickly seeing your code running in the cloud."?